src/computer_player.c

```diff
@@ -512,6 +513,7 @@ void process_playable_card(Card_t card)
     
     end_turn(COMPUTER);
     if (card.name == SKIP) {
+        print_info("COMPUTER discarded a Skip card, HUMAN will lose turn.\n");
```

Comment:
Can we change this to " Computer has dropped a skip card now, human player loses his turn" ?

Reply:
1. COMPUTER and HUMAN are specified for the player "name", just want to keep it.  2. avoid to use his, or we need to add her as well.

Reply:
Updated.